LocalSettings.php

```diff
@@ -1860,20 +1860,6 @@
 			'eswiki',
 			'wikispecies',
 		],
-		'+hispanowiki' => [
-			'u',
-			'w',
-		],
-		'+privadowiki' => [
```

Comment:
privadowiki was not mentioned in T6444

Reply:
I didn't see it necessary, my wiki doesn't need that configuration anymore and I wanted to take advantage of it. I've done something like this before but forgot to mention it when I opened this change.